src/node_snapshotable.cc

```diff
@@ -15,15 +15,25 @@
 #include "node_v8.h"
 #include "node_v8_platform-inl.h"
 
+#if HAVE_INSPECTOR
+#include "inspector/worker_inspector.h"  // ParentInspectorHandle
+#endif
+
```

Comment:
```suggestion
```
Is this required? ParentInspectorHandle is not being used in this file.

Reply:
Oh, I probably lost my reply somehow - this was necessary for the InitializeInspector() call below which takes a ParentInspectorHandle unique_ptr.

Reply:
Ah, the {} initializer, gotcha! 🤦